examples/tox21/predictor.py

```diff
@@ -78,8 +86,9 @@ def predict(self, atoms, adjs):
 class InferenceLoop(object):
     """Wrapper for predictors that offers inference loop."""
 
-    def __init__(self, predictor):
+    def __init__(self, predictor, batchsize=128):
         self.predictor = predictor
```

Comment:
I feel it is kind to check whether predictor has 'predict' method.
Like `callable(getattr(predictor_, 'predict', None))`
How do you think?

Reply:
I think so.